content/blog/getting-started-with-the-hpe-data-services-cloud-console-public-rest-api.md

```diff
@@ -0,0 +1,198 @@
+
+Some of the advantages of distributing the API in OpenAPI 3.0 format:
+
+1. Updates to the API can be generated in a more agile manner where documentation is embedded, describing any endpoints, parameters, and more; such as contact information, license, terms of use.
+2. Consumers of this API also gain the benefits of agility, by using the converter from openAPI yaml, or json to any programming language that is used as part of their automation or CI/CD workflow. (Please check https://openapi.tools for more information for the API tools to generate client code)
+
+### DSCC REST API Details
+
+Anyone can download this OpenAPI Specification (OAS) v3 definition of the DSCC from the following: [Link to the API repository](https://console-us1.data.cloud.hpe.com/doc/api/v1/)
+
+![HPE GreenLake API documentation](/img/api-documentation-display.png "API Doc")
+
+The website also provides additional information:
+
+1. The list of the REST API resources that are supported as of the release.
+2. The information about the HTTP method, parameters and the responses that are expected from each resource.
+3. The syntax for the HTTP method and path to this resource. Note that this path is a relative path. For the complete path, please add the base-URL documented below.
+4. The body of response is returned in JSON format according to the response status of the REST API.
+   The website also provides the links to download the DSCC OpenAPI definitions in either json or yaml format. Below is an example of the downloaded yaml definition file from the DSCC REST API documentation website.
+
+![API definition yaml](/img/open-api-yaml.png "yaml")
+
+User can download the API definition from the API documentation website, and the API definition is available in both YAML and JSON version. It can be downloaded by clicking on the download button on the top left of the documentation website.
+
+### Documented Attributes
+
+* API Name & Description
+
+  * Provides short description of the objective for this API with the supported HTTP request method (POST, GET, DELETE, PATCH, PUT etc).
+* API Path
+
+  * Provides the detailed URL path as the end-point to issue the API call. Note that the user must add the base path URL to extend this path to the correct resource end-point.
+* API Parameter
+
+  * Allows the client to input information such as the object for manipulation, select a filter to limit the returned objects, and other purposes.
+* API Data/Body/Payload
+
+  * This is the data passed along in a different part of the REST API request, usually associated with HTTP method such as POST/PATCH/PUT.
+* API Response
+
+  * Provides detail response information on the result of the particular API and may include more data in JSON format.
```

Comment:
Change "detail" to "detailed" so it reads "Provides detailed response information"